### DISCUSSION TITLE
Has Valinet Stopped Development of Explorer Patcher ?

### DISCUSSION THREAD

**Discussion by mustafatoktas** [valinet/ExplorerPatcher#3397] [Q&A] [answered]
My personal opinion is that valinet has stopped developing explorer patcher. It has been 2 months since the last stable version was released. However, despite all the bugs, a new stable version has still not been released.

A new bug is reported almost every day. You can see this in the issues tab.

Do you have any ideas about this?

We love Explorer Patcher, we use it and want its development to continue.

Note: pyrates999 if you are reading this do not reply or write anything.  I don't need your answers.

**Comment by Gunhedo** [comment]
I was also wondering, if this project has been abandoned.

**Reply by Amrsatrio** [reply, marked as answer]
No. I am currently working hard on getting the 10 taskbar to work again under 24H2, and left with little time to work on other parts of EP. Latest prerelease 65.5 works fine on the latest 22h2/23h2 build.

**Reply by Gunhedo** [reply]
Thank you for letting us know! 😊

**Reply by mistertheloaf** [reply]
> No. I am currently working hard on getting the 10 taskbar to work again under 24H2, and left with little time to work on other parts of EP. Latest prerelease 65.5 works fine on the latest 22h2/23h2 build.

You are not Valinet.  Where is Valinet?

**Reply by pyrates999** [reply]
There is more then one developer here.

**Comment by NormanFPC** [comment]
Windows 10taskbar can't be brought back so one fix for the next EP release is to make the Taskbar Overflow menu actually work by allowing drag and drop and rearranging program icons.

Useless in just displaying a row of program icons.

**Reply by pyrates999** [reply]
do you mean the system tray?

**Reply by mustafatoktas** [reply]
I think he means the system tray. I get it in that way

**Reply by pyrates999** [reply]
If it's the system tray, you can drag and drop the icon in it to the popup menu that you don't want shown.

this is already known about here: https://github.com/valinet/ExplorerPatcher/issues/1454
